documentation/docs/pmm-upgrade/migrating_from_pmm_2.md

```diff
@@ -113,6 +113,39 @@ Before upgrading to PMM 3, ensure your PMM 2 Server is running the latest versio
 Depending on your initial installation method, update PMM Clients using your operating system's package manager or by updating from a tarball.
 For detailed instructions, see the [Upgrade PMM Client topic](../pmm-upgrade/upgrade_client.md).
 
+## Step 4: Migrate your API keys to service accounts
+
+PMM 3 replaces API keys with service accounts to enhance security and simplify access management. You can trigger this API key conversion from the UI or from the CLI.
+	
+### From the UI:
+PMM automatically migrates existing API Keys to Service Accounts when you first log in as an Admin user. The migration results are displayed in a popup dialog box. If no popup appears, it likely means there are no API Keys to migrate—this is typical for PMM Servers without connected services.
+	
+### From CLI
+You can also trigger the conversion with the following command, replacing `admin:admin` with your credentials:
```

Comment:
Or user can has different address and port. Instead of 127.0.0.1 and 3000. Maybe we can mention also this.